ui/package.json

```diff
@@ -18,9 +18,9 @@
     "react-intl": "^2.8.0",
     "react-json-view": "1.19.1",
     "react-redux": "^7.1.0",
-    "react-router": "^4.3.1",
-    "react-router-dom": "^4.3.1",
-    "react-scripts": "^3.0.1",
+    "react-router": "^5.0.1",
+    "react-router-dom": "^5.0.1",
+    "react-scripts": "3.1.1",
```

Comment:
No `^`?

Reply:
For this specific update. I choose to update like in the [CHANGELOG](https://github.com/facebook/create-react-app/blob/master/CHANGELOG.md).
We can keep the `^`, that also fine. It's very unlikely that futures versions will break our app since `CRA` and `react` team in general are very careful when they introduce breaking change.